big-practice/src/js/pages/dashboard.page.js

```diff
@@ -222,11 +259,13 @@ class Dashboard {
   //   hideModal(modalWarning);
   // });
 
-  // // Movie will be updated when the update button is clicked
-  // modalFormUpdateBtn.addEventListener('click', () => {
-  //   handleUpdateForm();
-  //   hideModal(modalWarning);
-  // });
+  handleModalFormUpdateBtn() {
```

Comment:
handleModalFormUpdateBtn is a complicated name, could you use another simple name?

Reply:
Fixed